jsonnet/jsonnet.go

```diff
@@ -0,0 +1,53 @@
+package jsonnet
+
+import (
+	"context"
+	"fmt"
+	"io/fs"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/google/go-jsonnet"
+	"github.com/google/go-jsonnet/linter"
+	"github.com/grafana/scribe/plumbing/pipeline"
+)
+
+func Lint(path string) pipeline.Step {
+	path, err := filepath.Abs(path)
+	if err != nil {
+		return pipeline.NewStep(nil)
+	}
+	vm := jsonnet.MakeVM()
+	return pipeline.NewStep(
+		func(ctx context.Context, opts pipeline.ActionOpts) error {
+			return filepath.WalkDir(path, func(path string, d fs.DirEntry, err error) error {
```

Comment:
And instead, here we can set up the path to be relative to the source directory argument. I think we can use `:=` here because we're higher in scope.

```suggestion
		func(ctx context.Context, opts pipeline.ActionOpts) error {
			path := filepath.Join(path, opts.State.MustGetDirectoryString(pipeline.ArgumentSourceFS)
			return filepath.WalkDir(path, func(path string, d fs.DirEntry, err error) error {
```